cypress.config.js

```diff
@@ -1,9 +1,26 @@
+const { faker } = require('@faker-js/faker');
 const { defineConfig } = require('cypress');
 
 module.exports = defineConfig({
   e2e: {
+    baseUrl: 'https://demoqa.com',
     setupNodeEvents(on, config) {
-      // implement node event listeners here
+      on('task', {
+        generateWorkers() {
+          const workers = [];
+          for (let i = 0; i < 5; i++) {
+            workers.push({
+              firstName: faker.person.firstName(),
+              lastName: faker.person.lastName(),
+              email: faker.internet.email(),
+              age: faker.number.int({ min: 18, max: 100 }),
+              salary: faker.finance.pin(),
```

Comment:
The use of `faker.finance.pin()` for generating a salary might not be appropriate. `faker.finance.pin()` generates a PIN number, which is typically a 4-digit number. Consider using `faker.finance.amount()` or another suitable method to generate a realistic salary value.